src/test/compiled_ram_index_utils_test.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
```

Comment:
I am concerned that we use a different license for Patrick's code. Can we unify the license? Otherwise we end up in the license hell. I am not a lawyer but this will cause serious trouble in future. 

@pnappa  - you can dual license your source code, i.e., you can use the UPL license (which is a BSD style license) for the soufflé code, and your own license for your current-list GitHub repo. This is not a problem at all.